cfgov/paying_for_college/templates/paying_for_college/disclosure_feedback.html

```diff
@@ -25,19 +19,23 @@ <h1>Paying for College</h1>
       <div class="col col7 suf5">
           <hr>
           <header>
-              <h2>What do you think about this tool?</h2>
+              <h4>What do you think about this tool?</h4>
```

Comment:
What's the reason for this change? It will introduce a skipped-heading-levels accessibility error.

Reply:
I was just matching the prior design heading size, but I can keep it an h2 and it'll be larger.